corehq/blobs/s3db.py

```diff
@@ -184,7 +184,7 @@ def delete(self, *args, **kw):
     def bulk_delete(self, paths=None, metas=None):
         success = True
         s3_bucket = self._s3_bucket()
-        for chunk in chunked((paths if metas is None else metas), 1000):
+        for chunk in chunked((paths if metas is None else metas), 100):
```

Comment:
That's a big difference. What's the TCL limit? Do we need to go all the way down to 100?

Reply:
I ran a few tests on the cluster and above a few hundred it starts timing out. 100 seemed like a safe value.

Reply:
Does this mean deletes have really bad performance on TCL object storage? Maybe we should add a setting so we don't have to use such a low value on other envs where delete performance is better.